### DISCUSSION TITLE
Can I use teleport to connect o GCP cloudSQL without having a cloudSQL public IP?

### DISCUSSION THREAD

**Discussion by kopax-polyconseil** [gravitational/teleport#40899] [Q&A] [answered]
Hello,

I am reading :  

- https://goteleport.com/docs/ver/13.x/database-access/guides/postgres-cloudsql/
- https://goteleport.com/docs/access-controls/reference/
- https://goteleport.com/docs/ver/13.x/database-access/rbac/configuring-access/

I also have watched : https://www.youtube.com/watch?v=br9LZ3ZXqCk

As in the title, is it possible to setup a teleport SGBD rbac access if my GCP PostgreSQL instance does not have any public IP ?

Youtube demo, and doc state :  

> You will use the instance's public IP address to connect to it. It can be viewed on the "Connect to this instance" panel on the Overview page

<img width="739" alt="image" src="https://github.com/gravitational/teleport/assets/77674046/9fb920df-26e8-4806-b2b1-e0de1cae6a17">

But it is not clear if it is a requirement as it does not appears in the requirements section of the documentation.

**Comment by stevenGravy** [comment, marked as answer]
That's not required.  It just needs to be able to connect via the private or public address.
